lms/djangoapps/discussion/tests/test_views.py

```diff
@@ -356,11 +356,11 @@ def setUp(self):
         # course is outside the context manager that is verifying the number of queries,
         # and with split mongo, that method ends up querying disabled_xblocks (which is then
         # cached and hence not queried as part of call_single_thread).
-        (ModuleStoreEnum.Type.mongo, 1, 6, 4, 15, 3),
-        (ModuleStoreEnum.Type.mongo, 50, 6, 4, 15, 3),
+        (ModuleStoreEnum.Type.mongo, 1, 5, 3, 13, 1),
```

Comment:
We had some very bad performance regressions a couple of years ago, so these tests were put in to act as a baseline. IMO they are useful, and it is fairly infrequent that a change modifies the query count. It also prompts a discussion about query performance when it changes, which is generally a good thing. :-)